### PR TITLE
Trigger `selectRow` or `deselectRow` when a row is clicked.

### PR DESCRIPTION
**Pull request by stanislavgeorgiev**
Closes IBM/carbon-components-angular#393

### CODE REVIEW

**Comment by stanislavgeorgiev** [comment]
I added an `action()` logger but it doesn't get anything out... I know it gets executed because I tested it with a `console.log()`. I saw that you're using the action logger in other places and it does work fine there. Any suggestions?

**Comment by cal-smith** [comment]
Just going to close and reopen so travis will pick this up ...

**Comment by cal-smith** [comment]
I believe `action` returns a function and that would be passed to the `Output` ... the first parameter is just a name that it logs with ... it may work if you have `selectRow` just `return action("row selected");` (and so `(selectRow)="selectRow($event)"` would just become `(selectRow)="selectRow()"`), otherwise you'll need to move it into the story props and add an `Output` in the story component

**Comment by stanislavgeorgiev** [comment]
OK, I tried the `return action()` but it didn't change anything.
Anyways, that's not a hold up for the delivery I hope.

**Comment by stanislavgeorgiev** [comment]
What's the ETA on the second review and merging?

File: src/table/table.stories.ts [hunk 1]
```diff
@@ -37,8 +38,10 @@ import { clone } from "../utils/utils";
 			[model]="model"
 			[size]="size"
 			[showSelectionColumn]="showSelectionColumn"
+			[enableSingleSelect]="enableSingleSelect"
```
Comment (zvonimirfras) [inline src/table/table.stories.ts]: This is actually not officially supported by current carbon design so we're not going to surface it in the story.

Comment (stanislavgeorgiev) [reply src/table/table.stories.ts]: Sure, didn't know that. It's all reverted now except the fix.

**Pull request merged**